src/kernel/include/inv.h

```diff
@@ -90,6 +90,26 @@ sret_deactivate(struct cap_captbl *t, capid_t capin, livenessid_t lid)
 { return cap_capdeactivate(t, capin, CAP_SRET, lid); }
 
 static int
+asnd_construct(struct cap_asnd *asndc, struct cap_arcv *arcvc, capid_t rcv_cap, u32_t budget, u32_t period)
```

Comment:
We decided to keep it as FIXME because we cannot have captbl in this case to call asnd_activate.